### PR TITLE
chore: make export code snippet into a code block

### PR DESCRIPTION
**Pull request by axiomofjoy**
Dark Mode:

Screenshot 2024-01-09 at 2 02 35 PM

Light Mode:

Screenshot 2024-01-09 at 2 02 59 PM

### CODE REVIEW

**Comment by mikeldking** [comment]
Let's move the code block outside of the alert. That way it will be easy to see, we can add explainer text, and easier to add the clipboard icon

Screenshot 2024-01-09 at 10 30 45 AM

File: app/src/pages/embedding/ExportSelectionButton.tsx [hunk 1]
```diff
@@ -88,31 +127,27 @@ export function ExportSelectionButton() {
...
-                  `}
-                >
-                  <span>
-                    You can retrieve your export in your notebook via{" "}
-                  </span>
-                  <b>px.active_session().exports</b>
-                </p>
-              </div>
-            </Alert>
+                <span>You can retrieve your export in your notebook via </span>
```
Comment (mikeldking) [inline app/src/pages/embedding/ExportSelectionButton.tsx:147]:
```suggestion
                You can retrieve your export in your notebook via 
```

File: app/src/pages/embedding/ExportSelectionButton.tsx [hunk 2]
```diff
@@ -25,6 +30,40 @@ type ExportInfo = {
...
+  }
+`;
+function CodeBlock({ value }: { value: string }) {
+  const { theme } = useTheme();
+  const codeMirrorTheme = theme === "light" ? undefined : nord;
+  return (
+    <CodeMirror
+      value={value}
+      basicSetup={{
+        lineNumbers: true,
```
Comment (mikeldking) [inline app/src/pages/embedding/ExportSelectionButton.tsx:53]:
```suggestion
        lineNumbers: false,
```

File: app/src/pages/embedding/ExportSelectionButton.tsx [hunk 3]
```diff
@@ -25,6 +30,40 @@ type ExportInfo = {
   fileName: string;
 };
 
+const EXPORTS_CODE_SNIPPET = `import phoenix as px
+
+px.active_session().exports`;
```
Comment (mikeldking) [inline app/src/pages/embedding/ExportSelectionButton.tsx:35]: maybe add a line to show how to access the -1 and to view the dataframe

File: app/src/pages/embedding/ExportSelectionButton.tsx [hunk 4]
```diff
@@ -88,31 +127,27 @@ export function ExportSelectionButton() {
...
-                  <span>
-                    You can retrieve your export in your notebook via{" "}
-                  </span>
-                  <b>px.active_session().exports</b>
-                </p>
-              </div>
-            </Alert>
+                <span>You can retrieve your export in your notebook via </span>
+              </p>
+              <CodeBlock value={EXPORTS_CODE_SNIPPET} />
```
Comment (mikeldking) [inline app/src/pages/embedding/ExportSelectionButton.tsx:149]:
```suggestion
              <View borderColor="light" ... >
              <CodeBlock value={EXPORTS_CODE_SNIPPET} />
              </View>
```
I would add a border and some rounding here to contain the content nicely

**Review by mikeldking — APPROVED** [review]

File: app/src/pages/embedding/ExportSelectionButton.tsx (hunk 1 above)
Comment (axiomofjoy) [reply app/src/pages/embedding/ExportSelectionButton.tsx]: addressed in a1dd8447

File: app/src/pages/embedding/ExportSelectionButton.tsx (hunk 3 above)
Comment (axiomofjoy) [reply app/src/pages/embedding/ExportSelectionButton.tsx]: addressed in a1dd8447

File: app/src/pages/embedding/ExportSelectionButton.tsx (hunk 4 above)
Comment (axiomofjoy) [reply app/src/pages/embedding/ExportSelectionButton.tsx]: addressed in a1dd8447

**Pull request merged**